src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/FixedPolicy.cs

```diff
@@ -5,19 +5,17 @@
 
 namespace Azure.Base.Http.Pipeline
 {
-    class FixedPolicy : RetryPolicy {
-        int _maxRetries;
-        TimeSpan _delay;
-        int[] _retriableCodes;
+    public class FixedPolicy : RetryPolicy {
```

Comment:
FixedPolicy was a good name for an internal type. I think not that you made it public, it should be called FixedRetryPolicy. Having said that, why are you making it public?